<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cocinawindow</class>
 <widget class="QMainWindow" name="cocinawindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QListWidget" name="Comanda_2"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QListWidget" name="Comanda_7"/>
     </item>
     <item row="0" column="2">
      <widget class="QListWidget" name="Comanda_3"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QListWidget" name="Comanda1"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="Comanda_5"/>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="Comanda_6"/>
     </item>
     <item row="2" column="3">
      <widget class="QListWidget" name="Comanda_8"/>
     </item>
     <item row="0" column="3">
      <widget class="QListWidget" name="Comanda_4"/>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Terminado</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
